azure/template.json

```diff
@@ -1,31 +1,35 @@
 {
-    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
     "contentVersion": "1.0.0.0",
     "parameters": {
         "serviceName": {
             "type": "string"
         },
-        "aseHostingEnvironmentName": {
-            "type": "string",
-            "defaultValue": ""
-        },
-        "aseResourceGroup": {
-            "type": "string",
-            "defaultValue": ""
-        },
-        "appServicePlanName": {
+        "configurationStorageConnectionString": {
             "type": "string"
```

Comment:
securestring

Reply:
Changed `configurationStorageConnectionString` to securestring type.